sdk/keyvault/azure-security-keyvault-jca/README.md

```diff
@@ -122,6 +122,30 @@ azure:
     custom:         # The file location where you store the custom certificate
 ```
 
+### Key-Less certificates
+You can set the private key as [non-exportable] to ensure the security of the key.
+
+Note if you want to use key less certificate, you must add `sign` permission to the principal.
+![Sign To Principal](resources/SignToPrincipal.png)
```

Comment:
Could you please update here? Make it same to the starter module's readme.

Reply:
There is no description about creating resources and permissions in jca's readme. Do I need to add relevant content? Or just need to add a description about adding permissions?

Reply:
just need to add a description about adding permissions, please.

Reply:
I added some descriptions because there is no relevant content in the context. Can be deleted at any time

Reply:
OK.